sstables/proto/sstable.proto

```diff
@@ -22,4 +22,6 @@ message MetaData {
     uint64 totalBytes = 6;
     uint32 version = 7; // currently version 1, the default is version 0 with protos as values
     uint64 skippedRecords = 8;
+    uint64 tombRecords = 9;
```

Comment:
there's a makefile target for protos if you want to run it:
https://github.com/thomasjungblut/go-sstables/blob/main/Makefile#L18-L27

it's a bit finicky to get the versions right, I'm currently having:
```
$ protoc --version
libprotoc 27.2
```

feel free to checkin all the generated changes too

Reply:
will first try without making change to this protoc